library/xml-model/src/main/java/org/cru/godtools/xml/model/Manifest.kt

```diff
@@ -297,15 +309,14 @@ class Manifest : BaseModel, Styles {
                             val id = getAttributeValue(null, XML_TIPS_TIP_ID)
                             val src = getAttributeValue(null, XML_TIPS_TIP_SRC)
                             if (id != null && src != null)
-                                add(async { parseFile(src).use { Tip(this@Manifest, id, it) } })
+                                add(scope.async { parseFile(src).use { Tip(this@Manifest, id, it) } })
```

Comment:
Since this is using the scope passed in from the surrounding ```runBlocking```, the ```tipsTasks.awaitAll()``` above means it will wait for this to finish too right?

Reply:
yes, in the `parseTips()` function I start the process of parsing all the tips, and return a list of all of those `Deferred` tasks. then in the calling method I use `tipsTasks.awaitAll()` to wait for the tip parsing to finish